packages/atclient/src/atnotification.c

```diff
@@ -1907,3 +1579,340 @@ int atclient_atnotification_set_decrypted_value(atclient_atnotification *notific
   goto exit;
 exit: { return ret; }
 }
+
```

Comment:
Here's the new placement for the functions. We will need to implement `atclient_atnotification_from_json_str` for each json library we want to support.